src/AutorestSwift/View Models/ParameterViewModel.swift

```diff
@@ -41,7 +41,7 @@ struct ParameterViewModel {
         if let name = specificName, !name.isEmpty {
             self.name = name
         } else {
-            self.name = param.serializedName ?? param.name
+            self.name = /* param.serializedName ?? */ param.name
```

Comment:
Why is this commented out? Should it be removed?

Reply:
This change is outdated. And I check the latest iteration, it has been removed.